spec/features/regular_user_cant_delete_users_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.feature "RegularUserCantDeleteUsers", type: :feature do
```

Comment:
Same as above. I also like a name of the test more than admin version, it expresses precisely what this test should test.